crates/ruff_linter/resources/test/fixtures/ruff/RUF028.py

```diff
@@ -0,0 +1,50 @@
+def fmt_off_between_lists():
+    test_list = [
+        # fmt: off
+        1,
+        2,
+        3,
+    ]
+
+# note: the second `fmt: skip`` should be OK
+def fmt_skip_on_own_line():
+    # fmt: skip
+    pass # fmt: skip
+
+
+@fmt_skip_on_own_line
+# fmt: off
+@fmt_off_between_lists
+def fmt_off_between_decorators():
+    pass
+
+@fmt_off_between_decorators
+# fmt: off
+@fmt_off_between_lists
+class FmtOffBetweenClassDecorators:
+    ...
+
+def fmt_off_in_else():
+    x = [1, 2, 3]
+    for val in x:
+        print(x)
+    # fmt: off
+    else:
+        print("done")
+    while False:
+        print("while")
+        # fmt: off
+    # fmt: off
+    else:
+        print("done")
+    if len(x) > 3:
+        print("huh?")
+    # fmt: on
+    # fmt: off
+    else:
+        print("expected")
+
+def fmt_on_trailing():
+    # fmt: off
+    val = 5 # fmt: on
+    pass # fmt: on
```

Comment:
Can we add an example for a `fmt: skip` that is combined with another suppression comment (it works ;), just to have this tested:

```python
    def cls_method_a(
            # fmt: off
            cls) -> None: 
        # noqa: test # fmt: skip
        pass
```

Reply:
It seems like this fails because `# fmt: skip` is treated like it's supposed to be on its own line. Is that not supposed to be the case?

Reply:
Ah yeah sorry, The comment should be at the end of the function header

```python
    def cls_method_a(
            cls) -> None:  # noqa: test # fmt: skip
        pass

```